examples/plot_acq_func_examples.ipynb

```diff
@@ -0,0 +1,481 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "28573d04",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "import os \n",
+    "# current working directory\n",
+    "path = os.getcwd()\n",
+    "# parent directory\n",
+    "parent = os.path.join(path, os.pardir)\n",
+    "sys.path.append(os.path.abspath(parent))\n",
+    "from bayes_opt1 import BayesianOptimization\n",
+    "from bayes_opt1 import UtilityFunction\n",
+    "from plot_gp_function import plot_gp\n",
+    "import numpy as np\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib import gridspec\n",
+    "%matplotlib inline\n",
+    "\n",
+    "from scipy.stats import norm\n",
+    "from scipy.optimize import minimize\n",
+    "from plot_gp_function import plot_convergence"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a36cdfba",
+   "metadata": {},
+   "source": [
+    "# Comparative analysis on acquisition functions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "f27a22bd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x = np.linspace(-2, 10, 10000).reshape(-1,1)\n",
```

Comment:
`x` è un nome fuorviante per un vettore di punti. Potreste chiamarlo `xx`

Reply:
Abbiamo provato a cambiarlo ma partono una serie di errori :(

Reply:
Quali errori vi partono? È molto strano, dovrebbe funzionare senza alcun problema. Anzi, ho appena provato personalmente ed è tutto a posto. Notate che parlo solo della variabile `x` in questo notebook, non della stringa `'x'` nell'inizializzazione degli optimizer